pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -33,21 +32,23 @@ rustPlatform.buildRustPackage rec {
       "cuid-1.3.2" = "sha256-qBu1k/dJiA6rWBwk4nOOqouIneD9h2TTBT8tvs0TDfA=";
       "graphql-parser-0.3.0" = "sha256-0ZAsj2mW6fCLhwTETucjbu4rPNzfbNiHu2wVTBlTNe4=";
       "mysql_async-0.31.3" = "sha256-2wOupQ/LFV9pUifqBLwTvA0tySv+XWbxHiqs7iTzvvg=";
-      "postgres-native-tls-0.5.0" = "sha256-UYPsxhCkXXWk8yPbqjNS0illwjS5mVm3Z/jFwpVwqfw=";
+      "postgres-native-tls-0.5.0" = "sha256-4CftieImsG2mBqpoJFfyq0R2yd2EyQX4oddAwyXMDZc=";
+      "mongodb-3.0.0" = "sha256-1WQgY0zSZhFjt1nrLYTUBrpqBxpCCgKRSeGJLtkE6pw=";
     };
   };
 
-  nativeBuildInputs = [ pkg-config git ];
+  nativeBuildInputs = [ pkg-config ];
```

Comment:
`git` was removed because it wasn't usable for what the build scripts were trying to do (running `git rev-parse HEAD`). It used to silently fail before https://github.com/prisma/prisma-engines/pull/4986 and store an empty string instead of the commit hash, now it fails with a build error, so we make use of the `GIT_HASH` env var introduced in https://github.com/prisma/prisma-engines/pull/4992 instead.